apps/ehr/test/e2e/specs/inPersonVisitMainFlow.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { Page, test } from '@playwright/test';
+import { ResourceHandler } from '../../e2e-utils/resource-handler';
+import { ENV_LOCATION_NAME } from '../../e2e-utils/resource/constants';
+import { openVisitsPage } from '../page/VisitsPage';
+import { expectPatientInfoPage, PatientInfoPage } from '../page/PatientInfo';
+import { expectProgressNotePage } from '../page/ProgressNotePage';
+import { expectAssessmentPage } from '../page/AssessmentPage';
+
+const DIAGNOSIS = 'Situs inversus';
+const EM_CODE = '99201 New Patient - E/M Level 1';
+const resourceHandler = new ResourceHandler();
+
+test.beforeAll(async () => {
+  await resourceHandler.setResources();
+});
+
+test.afterAll(async () => {
+  await resourceHandler.cleanupResources();
+});
+
+test('Book appointment, start and complete Intake, check statuses', async ({ page }) => {
+  const patientInfoPage = await intakeTestAppointment(page);
+  await patientInfoPage.cssHeader().verifyStatus('intake');
+  await patientInfoPage.sideMenu().clickCompleteIntakeButton();
+  await patientInfoPage.cssHeader().verifyStatus('ready for provider');
+});
+
+test('Book appointment, go to Hospitalization page and complete Intake, check statuses', async ({ page }) => {
+  const patientInfoPage = await intakeTestAppointment(page);
+  const hospitalizationPage = await patientInfoPage.sideMenu().clickHospitalization();
+  await hospitalizationPage.clickCompleteIntakeButton();
+  await patientInfoPage.cssHeader().verifyStatus('ready for provider');
+});
+
+test('Book appointment, click Provider on "Patient info", check statuses', async ({ page }) => {
+  const patientInfoPage = await intakeTestAppointment(page);
+  await patientInfoPage.cssHeader().clickSwitchStatusButton('provider');
+  await patientInfoPage.cssHeader().verifyStatus('provider');
+});
+
+test('Book appointment,fill required fields for screening, review and sign progress note', async ({ page }) => {
```

Comment:
Screening is like "intake" here that is performed by nurse, which is not tested in this test

```suggestion
test('Book appointment,fill required fields for signing the visit, review and sign progress note', async ({ page }) => {
```

Reply:
Done.